pagefaultgames/pokerogue
# [Misc] Remove extra sprites

## What are the changes?
Sprites were deleted that do not accurately reflect the Pokemon their name would suggest, and would not have any reason to ever be used.

## Why am I doing these changes?
Their existence was noted [here](https://discord.com/channels/1125469663833370665/1229465263532019712/1263650854687346699).
They look like this:
![image](https://github.com/user-attachments/assets/d20c996c-ef0a-45b4-b82b-756680955a96)

So actually these shouldn't even be loaded, since Oinkologne has *Forms*, due to gameplay differences, instead of only a slightly different sprite. As a result, the files here aren't in a place the game would ever check. That being said, it's probably in our best interests to prevent these from ever being used, even accidentally in the future. 

## What did change?
Deleted files that I don't see any purpose for.

### Screenshots/Videos
No gameplay differences.

## How to test the changes?
I guess one could use overrides and savefiles to ensure that this is neither in the game currently, nor somehow after it's been deleted, if they really wanted to.

## Checklist

~~- [ ] Are the changes visual?~~
~~  - [ ] Have I provided screenshots/videos of the changes?~~
